spel/minimal-linux.json

```diff
@@ -209,6 +209,26 @@
             "temporary_security_group_source_cidrs": "{{ user `security_group_cidrs` }}",
             "type": "amazon-ebs",
             "user_data_file": "{{ template_dir }}/userdata/tmpfsroot-el8.cloud"
+        },
+        {
+            "flavor": "{{ user `openstack_flavor_name` }}",
+            "floating_ip_network": "{{ user `openstack_floating_ip_network_name`}}",
+            "image_name": "{{ user `spel_identifier` }}-{{ build_name }}-{{ user `spel_version` }}.x86_64-gp2",
+            "insecure": "{{ user `openstack_allow_insecure_connection` }}",
+            "name": "minimal-centos-7-openstack",
```

Comment:
Similar question as for the `gp2` question. Does OpenStack have different "hardware", "hypervisor", etc. types that might be a useful naming-component?

* If you look at the AWS builders' `name` values, we have "hvm" because, when we first stood up the project, AWS had PVM and HVM instance-types that had specific needs (that the `name`-value helped "explain").
* Similarly for Azure builders' `image`/`vhd` types in the name

Reply:
I _think_ OpenStack supports xen (PVM) as a hypervisor, but we currently plan to use KVM/Qemu (HVM) exclusively.  I tried to make the `name` field follow the naming convention of the other builders:
- minimal-centos-7-virtualbox
- minimal-centos-7-hvm
- minimal-rhel-7-hvm
- minimal-centos-7-azure-vhd
- minimal-centos-7-azure-image
- minimal-centos-8-hvm
- minimal-rhel-8-hvm

Now that I'm looking at it, We *might* need separate images for OpenStack Nova-based storage vs. OpenStack Cinder-based storage (which might look like Azure's image vs. vhd, maybe).  I'm not 100% certain though, so I'd lean toward the current name.  If you prefer, I can use `minimal-centos-7-openstack-image` now and eventually add `minimal-centos-7-openstack-vhd` when I get Cinder online.

Reply:
Cool. Like I say, my questions come from ignorance of OpenStack. 

Right now our naming isn't exactly "uniform": when the project was first conceived, we hadn't anticipated usage in other contexts, thus `-hvm` rather than `-aws-hvm`. Azure was the first alternate cloud-context added to the project. Wanted to make sure that OpenStack didn't have multiple contexts that might cause naming collisions or overlaps.

Going to leave this one as unresolved pending you telling us whether I'm overthinking or that you want to update the naming-tokens. =)

Reply:
I would also recommend `minimal-centos-7-openstack-image` in this case, please.

Reply:
Done (https://github.com/plus3it/spel/pull/439/commits/0e2e7844b8c82db7ce3285a57e7fba6734a94096).